tests/StubsTypeHintsTest.php

```diff
@@ -262,7 +262,7 @@ public static function testSignatureTypeHintsComplainPhpDocInMethods(PHPFunction
         $unifiedPhpDocTypes = array_map(function (string $type) {
             $typeParts = explode('\\', $type);
             $typeName = end($typeParts);
-            return preg_replace('/\w+\[]/', 'array', $typeName);
+            return preg_replace(['/\w+\[]/','#array<.*>#'], 'array', $typeName);
```

Comment:
a little comment would help quicker understand why there is such specific replacements :)